src/integrationCapture.ts

```diff
@@ -130,14 +148,29 @@ export default class IntegrationCapture {
     public capture(): void {
         const queryParams = this.captureQueryParams() || {};
         const cookies = this.captureCookies() || {};
+        const localStorage = this.captureLocalStorage() || {};
 
         // Exclude _fbc if fbclid is present
         // https://developers.facebook.com/docs/marketing-api/conversions-api/parameters/fbp-and-fbc#retrieve-from-fbclid-url-query-parameter
         if (queryParams['fbclid'] && cookies['_fbc']) {
             delete cookies['_fbc'];
         }
 
-        this.clickIds = { ...this.clickIds, ...queryParams, ...cookies };
+        // If both rtid and RoktTransactionId are present, prioritize rtid
+        // If RoktTransactionId is present in both cookies and localStorage,
+        // prioritize localStorage
```

Comment:
I can't remember if you said there were actually docs for this, or if this was shared internally from Ben.  If there are docs, I'd link similar to line 153-154 above